CONFIGURE.md

```diff
@@ -59,7 +59,7 @@ When built for use on a single machine, the library will only include the best a
 
 ## OQS_USE_OPENSSL
 
-This can be set to `ON` or `OFF`. When `ON`, the additional options `OQS_USE_AES_OPENSSL`, `OQS_USE_SHA2_OPENSSL`, and `OQS_USE_SHA3_OPENSSL` are made available to control whether liboqs uses OpenSSL's AES, SHA-2, and SHA-3 implementations. By default, `OQS_USE_AES_OPENSSL` and `OQS_USE_SHA2_OPENSSL` are `ON` while `OQS_USE_SHA3_OPENSSL` is `OFF`.
+This can be set to `ON` or `OFF`. When `ON`, the additional options `OQS_USE_AES_OPENSSL`, `OQS_USE_SHA2_OPENSSL`, and `OQS_USE_SHA3_OPENSSL` are made available to control whether liboqs uses OpenSSL's AES, SHA-2, and SHA-3 implementations. By default, `OQS_USE_AES_OPENSSL` is `ON` if `OQS_DIST_X86_64_BUILD` and `OQS_USE_AES_INSTRUCTIONS` are not set, `OQS_USE_SHA2_OPENSSL` is `ON` while `OQS_USE_SHA3_OPENSSL` is `OFF`.
```

Comment:
Can this be facilitated/explained using `OQS_DIST_BUILD` (documented variable) instead of `OQS_DIST_X86_64_BUILD` (i.e., what happens on ARM -- where `OQS_DIST_X86_64_BUILD` surely is not set)? Otherwise, there should be documentation for `OQS_DIST_X86_64_BUILD`. `OQS_USE_AES_INSTRUCTIONS` also is not (yet) documented in [CONFIGURE.md](https://github.com/open-quantum-safe/liboqs/blob/main/CONFIGURE.md).

Reply:
I'd rephrase it to "`OQS_DIST_BUILD` on x86_64" and add a section describing the options to (un)set CPU features.

Reply:
(updated in 0b91245833c1bb9756519316fb13b8cb4d67d2cd)